### ISSUE TITLE
Password used for reset page is not checked for failure (No Error, No Restart)

### ISSUE THREAD

**Issue by Whomiga** [edgan/att-fiber-gateway-info#14]
Noticed this in a previous version and just verified in Latest version
No error is returned and the gateway is not actually restarted using the wrong password
for the action of restart-gateway (I expect all resets will act this way)

**Comment by edgan** [comment]
Given that the changes in 1.0.15 were only a new flag, I am pretty sure that is isn't the version that introduced the problem. Which means the version previous to that isn't the known good.

Can you tell me which version works for you, and which version doesn't work for you?

To clarify, the restart function works, but fails to report a bad password?

**Comment by Whomiga** [comment]
If a wrong password is given, no reset function works, but it does not report the password as being bad, have to physically look at the gateway to see nothing occurred. And no version I've seen has given a password error if a bad password is used while trying to reset. The reset functions do not call the GetPath() function to get a password error.

**Comment by edgan** [comment]
Ok, I think the issue is straight forward enough. I can fix it.

But I still want to clarify, when the correct password is given, the restart works, right?

**Comment by Whomiga** [comment]
Yes, if the right password is given it does work.

**Comment by edgan** [comment]
I have released 1.0.16 to address your issue. I also added tests to catch like issues in the future.

Tell me if it works for you.

How the test is run for `restart-gateway`:
```
att-fiber-gateway-info -action restart-gateway -password 1234567890 -fresh
```
